demos/scripts/robot_kinematics_control_loop.cpp

```diff
@@ -0,0 +1,88 @@
+#include "state_representation/space/cartesian/CartesianPose.hpp"
+#include "state_representation/space/cartesian/CartesianTwist.hpp"
+#include "state_representation/robot/JointPositions.hpp"
+#include "state_representation/robot/JointVelocities.hpp"
+#include "dynamical_systems/Linear.hpp"
+#include "robot_model/Model.hpp"
+#include <chrono>
+#include <thread>
+#include <utility>
+
+using namespace state_representation;
+using namespace dynamical_systems;
+using namespace robot_model;
+using namespace std::chrono_literals;
+
+namespace {
+class DummyRobotInterface {
+private:
+  Model robot_; ///< robot model to compute the kinematics
+
+public:
+  JointState joint_state_; ///< current joint state of the robot
+  CartesianState eef_state_; ///< current eef state of the robot
```

Comment:
in our style guide, public members are not suffixed with `_` 
(but, if you disagree with this choice, we can debate it with regards to the style guide)

```suggestion
  JointState joint_state; ///< current joint state of the robot
  CartesianState eef_state; ///< current eef state of the robot
```

Reply:
Right no indeed this makes a lot of sense. I actually made them private first then forgot to change that.